src/Codeception/Module/Symfony.php

```diff
@@ -12,80 +12,67 @@
 use Symfony\Component\VarDumper\Cloner\Data;
 
 /**
- * This module uses Symfony Crawler and HttpKernel to emulate requests and test response.
+ * This module makes it possible to access Symfony features. It can be loaded in two ways:
+ * * Complete: Requests are emulated through Symfony's [DomCrawler](http://symfony.com/doc/current/components/dom_crawler.html) and [HttpKernel](https://symfony.com/doc/2.7/components/http_kernel.html)
+ * * Certain parts only: [PhpBrowser Module](http://codeception.com/docs/modules/PhpBrowser) or [WebDriver Module](http://codeception.com/docs/modules/WebDriver) is used for requests
```

Comment:
Ok, I got the idea but I don't like the proposed structure. You explain configuration from the technical point but not from point of a use case. 

Instead of "Complete" it should be called: "Functional Testing", with the configuration for functional tests.
Then explain how to load Symfony for acceptance tests with WebDriver or PhpBrowser.